test('Hello world!', () => {

})

test('error', () => {
    throw new Error('Failure!')
})